repository/concept.go

```diff
@@ -27,7 +31,54 @@ func InitRepository() *ConceptsRepositoryImp {
 	return conceptsRepository
 }
 
-func (r *ConceptsRepositoryImp) StoreConcept(ctx context.Context, update model.Update) error {
+func (r *ConceptsRepositoryImp) StoreConcept(ctx context.Context, update model.UpdateDbo) error {
 	_, err := r.collection.InsertOne(ctx, update, nil)
 	return err
 }
+
+func (r ConceptsRepositoryImp) GetConceptUpdates(ctx context.Context, query bson.D) ([]*model.UpdateDbo, error) {
+	current, err := r.collection.Find(ctx, query)
+	logrus.Info("Starting GetConceptUpdates")
+	if err != nil {
+		return nil, err
+	}
+	defer current.Close(ctx)
+	var updates []*model.UpdateDbo
+	for current.Next(ctx) {
+		var update model.UpdateDbo
+		err := bson.Unmarshal(current.Current, &update)
+		if err != nil {
+			return nil, err
+		}
+		updates = append(updates, &update)
+	}
+	if err := current.Err(); err != nil {
+		return nil, err
+	}
+	logrus.Info("Finished getting all concept updates from database")
+	return updates, nil
+}
+
+func (r ConceptsRepositoryImp) GetConceptUpdate(ctx context.Context, conceptId string) (*model.UpdateDbo, error) {
```

Comment:
```suggestion
func (r ConceptsRepositoryImp) GetConceptUpdate(ctx context.Context, conceptId string) (*model.UpdateDiff, error) {
```
Men ja, ta denne i ny pr